src/main/resources/config.yml

```diff
@@ -13,48 +13,97 @@ AssaultRequiredOwnersOnline: 1 # At least this many owners must be online for th
 AssaultOwnerWeightPercent: 1.0 # Percentage of each Owner's balance (under the maximum) to include in the total cost & damages (Can be greater than 100)
 AssaultMemberWeightPercent: 1.0 # Percentage of each Member's balance (under the maximum) to include in the total cost & damages (Can be greater than 100)
 AssaultDestroyableBlocks: # these blocks will be removed if destroyed in assault. Other blocks will not be removed, but still count for damages. Don't include craft flyblocks, or blocks with metadata, or you can have issues.
-  - 1
-  - 2
-  - 3
-  - 4
-  - 12
-  - 13
-  - 15
-  - 16
-  - 18
-  - 19
-  - 24
-  - 31
-  - 37
-  - 38
-  - 39
-  - 40
-  - 46
-  - 48
-  - 60
-  - 67
-  - 73
-  - 74
-  - 78
-  - 79
-  - 80
-  - 81
-  - 98
-  - 108
-  - 128
-  - 129
-  - 161
-  - 162
-  - 163
-  - 168
-  - 169
-  - 174
-  - 175
-  - 179
-  - 180
-  - 181
-  - 182
-  - 212
-
+  - stone
+  - diorite
+  - polished_diorite
+  - granite
+  - polished_granite
+  - andesite
+  - polished_andesite
+  - grass_block
+  - dirt
+  - coarse_dirt
+  - podzol
+  - cobblestone
+  - sand
+  - red_sand
+  - gravel
+  - gold_ore
+  - iron_ore
+  - oak_leaves
+  - jungle_leaves
+  - birch_leaves
+  - spruce_leaves
+  - sponge
+  - wet_sponge
+  - sandstone
+  - smooth_sandstone
+  - cut_sandstone
+  - chiseled_sandstone
+  - grass
+  - fern
+  - tall_grass
+  - large_fern
+  - dandelion
+  - poppy
+  - blue_orchid
+  - allium
+  - azure_bluet
+  - red_tulip
+  - orange_tulip
+  - white_tulip
+  - pink_tulip
+  - oxeye_daisy
+  - cornflower
+  - lily_of_the_valley
+  - wither_rose
+  - sunflower
+  - lilac
+  - rose_bush
+  - peony
+  - brown_mushroom
+  - red_mushroom
+  - tnt
+  - mossy_cobblestone
+  - farmland
+  - cobblestone_stairs
+  - redstone_ore
+  - snow
+  - ice
+  - snow_block
+  - cactus
+  - stone_bricks
+  - cracked_stone_bricks
+  - mossy_stone_bricks
+  - chiseled_stone_bricks
+  - brick_stairs
+  - stone_stairs
+  - diorite_stairs
+  - polished_diorite_stairs
+  - andesite_stairs
+  - polished_andesite_stairs
+  - granite_stairs
+  - polished_granite_stairs
+  - sandstone_stairs
+  - smooth_sandstone_stairs
+  - emerald_ore
+  - dark_oak_leaves
+  - acacia_leaves
+  - acacia_log
+  - dark_oak_log
+  - acacia_stairs
+  - dark_oak_stairs
+  - prismarine
+  - prismarine_stairs
+  - dark_prismarine
+  - dark_prismarine_stairs
+  - sea_lantern
+  - packed_ice
+  - red_sandstone
+  - red_smooth_sandstone
+  - red_sandstone_stairs
+  - red_smooth_sandstone_stairs
+  - sandstone_slab
+  - ice
```

Comment:
I actually prefer the change to material names, so I'd like to keep this here.  I would like them to be in ALL CAPS.

Reply:
It is also possible to call toUpperCase() too